From 1e767024edeaa2bc450649f346aa9af437bb258b Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:08:31 -0400
Subject: [tools] perf: Remove elide setup for SORT_MODE__MEMORY mode

Message-id: <1410181752-25631-267-git-send-email-jolsa@redhat.com>
Patchwork-id: 91976
O-Subject: [PATCH RHEL7.1 BZ1134356 266/307] perf tools: Remove elide setup for SORT_MODE__MEMORY mode
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 2ec85c628c4cecef0f82d177279c579aed0f9706
Author: Jiri Olsa <jolsa@kernel.org>
Date: Fri May 23 17:15:46 2014 +0200

description
===========
There's no need to setup elide of sort_dso sort entry again
with symbol_conf.dso_list list.

The only difference were list names of memory mode data,
which does not make much sense to me.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/sort.c b/tools/perf/util/sort.c
index 254f583..2aba620 100644
--- a/tools/perf/util/sort.c
+++ b/tools/perf/util/sort.c
@@ -1412,19 +1412,6 @@ void sort__setup_elide(FILE *output)
   sort_entry__setup_elide(&sort_sym_to,
      symbol_conf.sym_to_list,
      "sym_to", output);
- } else if (sort__mode == SORT_MODE__MEMORY) {
-  sort_entry__setup_elide(&sort_dso, symbol_conf.dso_list,
-     "symbol_daddr", output);
-  sort_entry__setup_elide(&sort_dso, symbol_conf.dso_list,
-     "dso_daddr", output);
-  sort_entry__setup_elide(&sort_dso, symbol_conf.dso_list,
-     "mem", output);
-  sort_entry__setup_elide(&sort_dso, symbol_conf.dso_list,
-     "local_weight", output);
-  sort_entry__setup_elide(&sort_dso, symbol_conf.dso_list,
-     "tlb", output);
-  sort_entry__setup_elide(&sort_dso, symbol_conf.dso_list,
-     "snoop", output);
  }
 
  /*
-- 
1.7.1